src/Documenter.jl

```diff
@@ -527,22 +537,22 @@ function git_push(
                         cd(dirname) do; symlink(devurl, "latest"); end
                     end
                 else
-                    @assert occursin(Base.VERSION_REGEX, tag) # checked in deploydocs
-                    version = VersionNumber(tag)
-                    # only push to stable if this is the latest stable release
-                    versions = filter!(x -> occursin(Base.VERSION_REGEX, x), readdir(dirname))
-                    maxver = mapreduce(x -> VersionNumber(x), max, versions; init=v"0.0.0")
-                    if version >= maxver && version.prerelease == () # don't deploy to stable for prereleases
-                        gitrm_copy(target_dir, stable_dir)
-                        Writers.HTMLWriter.generate_siteinfo_file(stable_dir, "stable")
-                    end
+                    tagged_dir = joinpath(dirname, tag)
                     gitrm_copy(target_dir, tagged_dir)
                     Writers.HTMLWriter.generate_siteinfo_file(tagged_dir, tag)
                 end
 
-                # Create the versions.js file containing a list of all docs
-                # versions. This must always happen after the folder copying.
-                Writers.HTMLWriter.generate_version_file(dirname)
+                # Expand the users `selector` vector
+                entries, symlinks = Writers.HTMLWriter.expand_selector(dirname, selector)
+
+                # Create the versions.js file containing a list of `entries`.
+                # This must always happen after the folder copying.
+                Writers.HTMLWriter.generate_version_file(dirname, entries)
+
+                # generate the symlinks
+                cd(dirname) do
+                    foreach(kv -> symlink(kv.second, kv.first), symlinks)
```

Comment:
Need to remove existing links here. And I guess error if what exist there is not an old symlink?